core/arch/arm/include/sm/psci.h

```diff
@@ -58,3 +58,5 @@ int psci_node_hw_state(uint32_t cpu_id, uint32_t power_level);
 int psci_stat_residency(uint32_t cpu_id, uint32_t power_state);
 int psci_stat_count(uint32_t cpu_id, uint32_t power_state);
 void tee_psci_handler(struct thread_smc_args *args);
+
+void psci_armv7_cpu_off(void);
```

Comment:
They are now only used by psci_armv7_cpu_off. We could export them when we need in future, i think:)